Cargo.toml

```diff
@@ -14,3 +14,10 @@ repository = "https://github.com/ikalnytskyi/youshallnotpass"
 license = "MIT"
 keywords = ["rate-limiter", "token-bucket"] 
 categories = ["algorithms", "data-structures"] 
+
+[dev-dependencies]
+criterion = "0.4.0"
+
+[[bench]]
+name = "benchmarks"
+harness = false  # disable the standard test harness, as we want to use the one provided by criterion
```

Comment:
How come `cargo bench` is built-in and stable while `libtest::Bencher` is experimental? :exploding_head: What would happen if we won't disable `harness`?

Reply:
I guess it's one of those things where they expect API changes and don't want to stabilize it yet :(

Disabling `harness` just ignores the benchmarks using `criterion`, so they won't be run.